pkg/storage/truncate.go

```diff
@@ -50,3 +50,10 @@ func (s *Store) TruncateAllTablesDANGEROUS(logger *zap.Logger) error {
 
 	return nil
 }
+
+// DeleteUserAccountDANGEROUS deletes a given user account from the DB
+// only to be used in test code as we do not truncate `user_account` table
```

Comment:
Any idea why we _don't_ truncate the `user_account` table? I would think operations like `scripts/dev db:seed` would simply re-create anything truncated.

Have you tried truncating it to see if anything fails? If not, I'd prefer just adding `user_account` to the list of tables we clear on tests rather than have to have tests handle their own cleanup.

Reply:
I'll answer my own question -- StackOverflow style!

Truncating the entire `user_account` table would also truncate the "default" users created in the initial migration [here](https://github.com/CMSgov/easi-app/blob/0acf5e409b637e3b7cde5923a6be16cf2dbb3ac3/migrations/V170__Add_user_account_table.sql#L17-L34), so that's likely why truncating the entire table isn't something we want to do!

Reply:
Exactly! 😄 I'll document that though - I should have! Thanks Clay